src/main/java/com/gluonhq/substrate/target/IosTargetConfiguration.java

```diff
@@ -112,6 +112,7 @@ public IosTargetConfiguration(ProcessPaths paths, InternalProjectConfiguration c
         Path llcPath = getLlcPath();
         return Arrays.asList("-H:CompilerBackend=" + Constants.BACKEND_LLVM,
                 "-H:-SpawnIsolates",
+                "-Dllvm.bin.dir=" + projectConfiguration.getGraalPath().resolve("lib/llvm/bin"),
```

Comment:
probably better to do `resolve("lib").resolve("llvm").resolve("bin")`? Do we need to check if that exists?